metadata-ingestion/src/datahub/ingestion/source/tableau/__init__.py

```diff
@@ -0,0 +1,6 @@
+from datahub.ingestion.source.tableau.tableau import (
```

Comment:
change the mapping in setup.py, and then this file should be empty

Reply:
OK, I thought to avoid tableau.tableau i.e. repetition of tableau word